tools/loris_db_upgrade.php

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/php
+<?php
+
+/**
+ * This script will upgrade the loris database to the lastest. 
```

Comment:
```suggestion
 * This script will upgrade the loris database to a specified release version.
```

Reply:
Agree @johnsaigle  that it should probably target a milestone/branch name, and this should be more explicit in terms of input/output